src/chat/ChatScreen.tsx

```diff
@@ -96,6 +102,14 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
     [firebaseInstance, memberIds, partners]
   );
 
+  const customInputToolbar = useCallback(
+    (composeProps: ComposerProps) => {
+      if (renderComposer) return renderComposer(composeProps);
+      return <InputToolbar {...composeProps} />;
```

Comment:
how to custom style for `InputToolbar`

Reply:
I added styleInputToolbar to custom this component